src/Dapr.Actors.AspNetCore/EndpointRouteBuilderExtensions.cs

```diff
@@ -7,32 +7,33 @@ namespace Dapr.Actors.AspNetCore
 {
     using System;
     using Dapr.Actors.Runtime;
+    using Microsoft.AspNetCore.Builder;
     using Microsoft.AspNetCore.Http;
     using Microsoft.AspNetCore.Routing;
 
-    internal static class RouterBuilderExtensions
+    internal static class EndpointRouteBuilderExtensions
     {
-        public static void AddDaprConfigRoute(this IRouteBuilder routeBuilder)
+        public static IEndpointConventionBuilder AddDaprConfigRoute(this IEndpointRouteBuilder endpoints)
         {
-            routeBuilder.MapGet("dapr/config", async context =>
+            return endpoints.MapGet("dapr/config", async context =>
             {
                 context.Response.ContentType = "application/json";
-                await ActorRuntime.Instance.SerializeSettingsAndRegisteredTypes(context.Response.BodyWriter);                
+                await ActorRuntime.Instance.SerializeSettingsAndRegisteredTypes(context.Response.BodyWriter);
             });
         }
 
-        public static void AddDaprHealthzRoute(this IRouteBuilder routeBuilder)
+        public static IEndpointConventionBuilder AddDaprHealthzRoute(this IEndpointRouteBuilder endpoints)
         {
-            routeBuilder.MapGet("healthz", async context =>
+            return endpoints.MapGet("healthz", async context =>
```

Comment:
If you want to btw you could change this to use the built-in health checks feature. Then the health checks are configurable instead of just always returning 200.

Reply:
Dapr runtime currently only checks for 200 OK

Reply:
it's not related to what's *returned* in the response. 

Health checks allows the healthy/unhealthy check to do something configurable in the app to make that decision. https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-3.1

I'm surprised that no one has asked for this yet 😆

Reply:
added endpoints.MapHealthChecks